app/controllers/api/v1/api_controller.rb

```diff
@@ -15,7 +15,8 @@ class ApiController < ApplicationController
       # Raises UnauthorizedRequestError exception on invalid token lookup
       # Returns a Tenant
```

Comment:
`Returns a Tenant` to `Returns a boolean and increments the authenticated tenant`

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/6/commits/68108ecaf0594493cd9e1acd245804e89f93f03b